backend/README.md

```diff
@@ -30,6 +31,20 @@ To change the ports of the application and various other launch settings (such a
 Read more about the `launchSettings.json` file
 [here](https://docs.microsoft.com/en-us/aspnet/core/fundamentals/environments?view=aspnetcore-6.0&preserve-view=true&viewFallbackFrom=aspnetcore-2.2#lsj)
 
+### Run in Docker
+
+For the backend to work when dockerized, you need to have the client secret exposed as
+an environment variable named `FlotillaClientSecret`.
+The simplest way to do this in bash is to run
+```
+export FlotillaClientSecret=SuperSecret
```

Comment:
Should we call it FLOTILLA_CLIENT_SECRET to be consistent with other secrets, such as ISAR_MQTT_PASSWORD?

Reply:
Agreed